docs/web3j-openapi.md

```diff
@@ -0,0 +1,248 @@
+# Web3j-OpenAPI
+
+[Web3j-OpenAPI](https://github.com/web3j/web3j-openapi) is an [OpenAPI](https://swagger.io/specification/) generator from [Solidity](https://solidity.readthedocs.io/) smart contracts. It provides a way to interact with the [Ethereum](https://ethereum.org/) blockchain via simple and intuitive HTTP requests. These interactions can be done using plain HTTP requests or via the [Swagger-UI](https://swagger.io/tools/swagger-ui/), which is generated with every project.
+
+This workflow can be summed in the following steps:
+- Writing a solidity smart contract
+- Generating the corresponding OpenAPI project using Web3j-OpenAPI
+- Running the generated project
+- Sending HTTP requests using Curls or Swagger-UI
```

Comment:
`Curl` not `Curls`

Reply:
In the 3rd line I would mention something more like:
> Running the generated project as a standalone server